packages/router/test/adapters/contract/contract.spec.ts

```diff
@@ -396,29 +405,27 @@ describe("Contract Adapter", () => {
     });
   });
 
-  describe("getRouterBalance", async () => {
+  describe("#getRouterBalance", async () => {
     it("should work", async () => {
       txServiceMock.readTx.resolves("10");
       const ret = await ContractFns.getRouterBalance(1337, routerAddrMock, mkAddress());
       expect(ret.toNumber()).to.be.eq(10);
     });
   });
 
-  describe("prepareRouterContract", async () => {
+  describe("#prepareRouterContract", async () => {
     beforeEach(() => {
       sanitationStub = stub(SharedFns, "sanitationCheck").resolves();
+      txServiceMock.sendTx.withArgs(onchainTxMock, requestContext).resolves(txReceiptMock);
     });
 
-    it("should work", async () => {
-      const chainId = txDataMock.sendingChainId;
-      const routerRelayerFeeAsset = mkAddress("0x00");
-
+    it("should work (without relayers)", async () => {
       const res = await ContractFns.prepareRouterContract(
-        chainId,
+        chainIdMock,
         prepareParamsMock,
         routerContractAddressMock,
         sigMock,
-        routerRelayerFeeAsset,
+        routerRelayerFeeAssetMock,
         "1",
```

Comment:
why leave out the router relayer fee from the constants tho

Reply:
lol not sure, didn't really think about it